spec/features/log_in_spec.rb

```diff
@@ -1,11 +1,14 @@
 require "rails_helper"
 
 feature "Log in" do
-  scenario "A valid email and password user is able to log in" do
-    email = "test@example.com"
-    password = "password"
+  given(:email)    { "test@examle.com" }
```

Comment:
I did a simple refactoring before identifying the cause. This is because I wanted to add cases and make it a particular help. It is a refactoring unrelated to this fix, should I undo it?